packages/slider/README.md

```diff
@@ -1,64 +1,124 @@
-# mwc-slider
+# `<mwc-slider>` [![Published on npm](https://img.shields.io/npm/v/@material/mwc-slider.svg)](https://www.npmjs.com/package/@material/mwc-slider)
 
-> :warning: These components are a work in progress. They are pre-release and should be considered experimental, as they may undergo major changes before release. We are experimenting with alternate architectures and approaches with the goal of allowing us to bring the most correct and optimal implementation of Material components to the widest possible audiences. Visible progress may be slow, as this research is across teams and repositories so is not consistently reflected in commits to this codebase. :warning:
+> IMPORTANT: The Material Web Components are a work in progress and subject to
+> major changes until 1.0 release.
 
-A [Material Components](https://material.io/components/) icon implementation using [Web Components](https://www.webcomponents.org/introduction)
+Sliders allow users to make selections from a range of values.
 
-## Getting started
+**Note: vertical sliders and range (multi-thumb) sliders are not supported, due to their absence from the material design spec.**
 
- * The easiest way to try out mwc-slider is to use one of these online tools:
+<img src="images/standard.png" width="397px">
```

Comment:
This image doesn't exist.

Also I think it's really *height* that we should specify in the img tags, not width, to prevent jank.